### PR TITLE
#1931: Allow sgcollect to collect basic info on Windows

### PR DESCRIPTION
**Pull request by mattcarabine**
This change is [Reviewable](https://reviewable.io/reviews/couchbase/sync_gateway/1954)

### CODE REVIEW

**Comment by tleyden** [comment]
@ajres I assigned you as reviewer

File: tools/sgcollect_info [hunk 1]
```diff
@@ -168,18 +171,27 @@ def make_collect_logs_tasks(zip_dir, sg_url):
...
     for sg_log_file, path in sg_log_files.iteritems():
-        task = AllOsTask(
+        task = UnixTask(
             "sg logs (%s)" % sg_log_file,
             "cat {}".format(path),
             log_file=sg_log_file,
         )
         sg_tasks.append(task)
 
+    sg_tasks.append(WindowsTask(
```
Comment (mattcarabine) [inline tools/sgcollect_info]: This needs changes to check if sg_log_path exists!

**Comment by mattcarabine** [comment]
I was doing some more testing on Windows and found that the upload functionality does not work on Windows (whereas it does on Linux)... will dig around to see if I can find out why!

**Comment by ajres** [comment]
PR #1955 service wrapper redirects stderr logs on Windows to "C:\Program Files (x86)\Couchbase\var\lib\couchbase\logs\sync_gateway_error.log"

**Pull request closed without merging**